lib/openstudio/workflow/jobs/run_energyplus/run_energyplus.rb

```diff
@@ -117,6 +112,45 @@ def perform
 
   private
 
+  # Look for the location of EnergyPlus
+  def find_energyplus
+    if ENV['ENERGYPLUSDIR']
+      return ENV['ENERGYPLUSDIR']
+    elsif ENV['RUBYLIB'] =~ /OpenStudio/
```

Comment:
I don't think we currently have this method in OpenStudio but I think that we should add it here: https://github.com/NREL/OpenStudio/blob/develop/openstudiocore/src/utilities/core/ApplicationPathHelpers.hpp

@kbenne is that what you are thinking too?